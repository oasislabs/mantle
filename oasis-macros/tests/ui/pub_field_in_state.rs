oasis_std::contract! {

#[derive(Contract, Default)]
pub struct Counter {
    pub count: u32
}

impl Counter {
    pub fn new(ctx: &Context) -> Self {
        Default::default()
    }
}

}

fn main() {}
