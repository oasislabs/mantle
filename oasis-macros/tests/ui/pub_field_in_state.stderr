warning: Field should have no visibility marker.
 --> $DIR/pub_field_in_state.rs:5:5
  |
5 |     pub count: u32
  |     ^^^

